src/tables/HVAR.js

```diff
@@ -0,0 +1,78 @@
+import r from 'restructure';
+import {resolveLength} from 'restructure/src/utils';
+
+
+let F2DOT14 = new r.Fixed(16, 'BE', 14);
```

Comment:
I switched to using `let` instead of `var` everywhere. I like the block scoping when used inside `if` statements etc., and for consistency I just use it everywhere now.